support for FT.GET, FT.MGET, FT.SUGLEN, FT.SUGDEL, FT.ALIASADD, FT.ALIASUPDATE, FT.ALIASDEL, FT.DICTADD, FT.DICTDEL, FT.DICTDUMP

filipecosta90 opened the pull request:
- [add] support for FT.GET and FT.MGET
- [add] support for FT.SUGLEN and FT.SUGDEL 
- [add] added default options for FT.SUGGET. 
- [add] add support for FT.DICTADD FT.DICTDEL and FT.DICTDUMP
- [add] added support for FT.ALIASADD, FT.ALIASUPDATE, FT.ALIASDEL
- [add] Increased coverage

filipecosta90 commented:
@mnunberg and @MeirShpilraien can you guys revise this one :) it extends the client support commands

itamarhaber commented on redisearch/autocomplete.go:
```diff
@@ -32,8 +36,7 @@ func (a *Autocompleter) Delete() error {
 // AddTerms pushes new term suggestions to the index
-func (a *Autocompleter) AddTerms(terms ...Suggestion) error {
-
+func (a *Autocompleter) AddTerms( terms ...Suggestion) error {
```
Extra space after opening paren

itamarhaber commented on redisearch/autocomplete.go:
```diff
@@ -62,49 +65,85 @@ func (a *Autocompleter) AddTerms(terms ...Suggestion) error {
 //
 // Deprecated: Please use SuggestOpts() instead
-func (a *Autocompleter) Suggest(prefix string, num int, fuzzy bool) ([]Suggestion, error) {
+func (a *Autocompleter) Suggest(prefix string, num int, fuzzy bool) (ret []Suggestion, err error) {
```
Since this is deprecated no real need to patch it up, but as you already have...

itamarhaber commented on redisearch/schema.go:
```diff
@@ -3,6 +3,31 @@ package redisearch
+
+	// If set, we will not save the documents contents, just index them, for fetching ids only
+	NoSave bool
+
```
Please complete the missing inline docs

filipecosta90 replied:
done. thanks for pointing this out

itamarhaber commented on redisearch/schema.go:
```diff
@@ -3,6 +3,31 @@ package redisearch
+	NoSave bool
+
+	NoFieldFlags bool
+
```
Please complete the missing inline docs

itamarhaber commented on redisearch/schema.go:
```diff
@@ -3,6 +3,31 @@ package redisearch
+	NoFieldFlags bool
+
+	NoFrequencies bool
+
```
Please complete the missing inline docs

filipecosta90 replied:
done. thanks for pointing this out

itamarhaber commented on redisearch/schema.go:
```diff
@@ -3,6 +3,31 @@ package redisearch
+	NoFrequencies bool
+
+	NoOffsetVectors bool
+
```
Please complete the missing inline docs

filipecosta90 commented:
@itamarhaber also added the inline docs for `IndexingOptions` and moved IndexingOptions from query.go to document.go since they represent the options for indexing a single document
